.circleci/config.yml

```
@@ -119,6 +119,9 @@ jobs:
       BUILD_FLAVOUR: validate
     steps:
       - checkout
+      - run:
+          name: Check CPU propertiese
```

```suggestion
          name: Check CPU properties
```
(though this step could be removed)

---

I removed these changes, they were only for debugging. We are going to disable Circle CI webhook once it is merged anyway.